runtime/onert/core/src/ir/DataType.cc

```diff
@@ -48,10 +49,12 @@ size_t sizeOfDataType(DataType data_type)
       return sizeof(int64_t);
     case DataType::QUANT_INT16_SYMM:
       return sizeof(int16_t);
-    default:
-      // ggml block quantize type data size is not supported
-      throw std::runtime_error{"Unsupported type size"};
+    case DataType::QUANT_GGML_Q4_0:
+    case DataType::QUANT_GGML_Q8_0:
+      // GGML block quantize type data size is not supported
```

Comment:
If the GGML data types are not supported I think they should fall into the `default` case as before. This change just lists them explicitly but all the previous cases return the size for the supported data types, while this one breaks. Plus the default is now gone which is a potential problem in static analysis tools. I'm not sure that I agree with this approach.

Reply:
On contrary, using default in most cases is problematic because static analysis can not verify that some keys are missing. IMHO, the best practice when dealing with enums is to list all keys explicitly (if possible) and handle "default" case outside of the `switch`. Then, in case when new enum value is added, compiler will report that not all values are handled - you don't have to run the code to see that some functionality is not implemented.